_usage/command-line.md

```diff
@@ -1,212 +1,121 @@
 ---
 layout: default
-title: Using the command line
+title: Connecting to Babelfish
 nav_order: 1
 ---
 
-## Using client tools to work with Babelfish
+## Using client tools to connect to Babelfish
 
-Most people use SQL Server Management Studio (SSMS) on Microsoft Windows to
-connect to Babelfish.
-
-However, there is an ever-growing community which prefers to use the command
-line to work with the server.
-Those users have the following command-line tool choices on Linux:
+On Windows, most people use SQL Server Management Studio (SSMS) or `sqlcmd` to
+connect to SQL Server, and these can be used to connect to Babelfish as well.
+SSMS is not available on Linux, so you are limited to using `sqlcmd` there.
```

Comment:
You can use a SQL Server client to connect with Babelfish on the TDS port. When connecting from a Windows system, SQL Server Management Studio (SSMS) or `sqlcmd` are good choices of clients. If you are connecting to the TDS port from a Linux system, `sqlcmd` is a good choice.

Reply:
Ok, done.